source/SAMRAI/pdat/SideData.h

```diff
@@ -151,28 +151,58 @@ class SideData:public hier::PatchData
       const hier::IntVector& directions);
 
 #if defined(HAVE_UMPIRE)
+   /*!
+    * @brief The constructor for a side data object.
+    *
+    * @param box const Box reference describing the interior of the
+    *            standard CELL-centered index box over which the
+    *            side data object will be created.
+    * @param depth gives the number of components for each
+    *              spatial location in the array.
+    * @param ghosts const IntVector reference indicating the width
+    *               of the ghost cell region around the box over which
+    *               the side data will be allocated.
+    * @param directions const IntVector reference indicating which
+    *                   coordinate directions will have data associated
+    *                   with them.
+    * @param allocator An umpire allocator to manage the allocation of the
```

Comment:
```suggestion
    * @param allocator An umpire::Allocator to manage the allocation of the
```

Reply:
I think we should use either `an Umpire allocator` or `an umpire::Allocator` for all of these comments.